java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -975,6 +992,8 @@ public void nextVector(ColumnVector previousVector,
       TimestampColumnVector result = (TimestampColumnVector) previousVector;
       super.nextVector(previousVector, isNull, batchSize);
 
+      // TODO: If context.isUseUTCTimestamp(), set TimestampColumnVector.useUTC to true
```

Comment:
I think it is better to set storage-api to 3.0.0 and fix this TODO in this patch as well.

Reply:
There is a vote going on for a storage-api release, then next week I can rebase the patch to consume it and hopefully we can check it in. Thanks @wgtmac !